clients/batch-stats.js

```diff
@@ -64,7 +64,6 @@ function handleStat(stat) {
         stat.name === 'tchannel.inbound.calls.recvd'
     ) {
         self.writeToStatsd(self.statsd.globalClient, stat);
-        self.writeToStatsd(self.statsd.perWorkerClient, stat);
```

Comment:
We want to keep this line.

I recommend we keep `recvd` per worker as it allows us to understand what QPS is going through a single HOT worker. this is a common debugging technique based on our alerts.

I recommend we remove `latency` for most service with the exception of the `hyperbahn` service being whitelisted, tracking the hyperbahn latency on a per-worker basis allows us to debug any issues with advertise.